### DISCUSSION TITLE
Kitty image protocol and scrolling

### DISCUSSION THREAD

**Discussion by aslpavel** [kovidgoyal/kitty#5522] [Q&A] [answered]
Is there a way to prevent kitty from scrolling the screen if an image is vertically larger than available space? Basically I would want to clip image, just as it happens when it is horizontally larger than terminal width.

**Comment by kovidgoyal** [comment, marked as answer]
Set the cursor movement policy C=1

**Reply by aslpavel** [reply]
Yes, that is exactly what I need. Thank you!
